src/ui/public/share/index.js

```diff
@@ -0,0 +1,57 @@
+define(function (require) {
```

Comment:
We should use CommonJS syntax instead of AMD, per styleguide: https://github.com/elastic/kibana/blob/master/STYLEGUIDE.md#commonjs-syntax